rpc/backend/src/main/scala/io/udash/rpc/AtmosphereService.scala

```diff
@@ -38,20 +42,27 @@ trait AtmosphereServiceConfig[ServerRPCType] {
   * @param config Configuration of AtmosphereService.
   * @tparam ServerRPCType Main server side RPC interface
   */
-class AtmosphereService[ServerRPCType](config: AtmosphereServiceConfig[ServerRPCType])
+class AtmosphereService[ServerRPCType](config: AtmosphereServiceConfig[ServerRPCType], sseSuspendTime: FiniteDuration = 1 minute)
                                       (implicit val executionContext: ExecutionContext)
   extends AtmosphereServletProcessor with LazyLogging {
 
-  override def init(config: AtmosphereConfig): Unit =
-    BroadcastManager.init(config.getBroadcasterFactory, config.metaBroadcaster())
+  private var brodcasterFactory: BroadcasterFactory = _
 
-  override def onRequest(resource: AtmosphereResource): Unit = {
-    config.initRpc(resource)
+  override def init(config: AtmosphereConfig): Unit = {
+    brodcasterFactory = config.getBroadcasterFactory
```

Comment:
Do we ever want to re-initialize this? Make it a `val` and init in constructor unless we do.